Applications/Custom/LayerClient/res/custom_layer_client.ini

```diff
@@ -26,3 +25,6 @@ Type = fully_connected
 unit = 10
 Bias_initializer = zeros
 Activation = softmax
+
+[losslayer]
+Type = mae_loss # custom loss layer
```

Comment:
i wonder if `loss = mae_loss` in `model` section also works?

Reply:
If yes, add as a comment here? if no, then it needs debugging.

Reply:
The design here is that
1. Custom loss is not supported from model section(except the traditional basic ones)
2. We support loss as normal layers that is `requireLabel() == true`

To support multi loss, we aggregates loss from `requireLabel() == true`

So it is no bug but intended

Reply:
> Custom loss is not supported from model section(except the traditional basic ones)

What about a custom loss from .so?

Reply:
Either from .so or directly from there source code which implements layer_devel.h

I'll update the comment inside ini file to cover the scenario you mentioned